hugo/layouts/papers/single.html

```diff
@@ -19,6 +19,7 @@
 {{ end }}
 {{ with $paper.pdf }}
   <meta content="{{ . }}" name="citation_pdf_url" >
+  <link rel="canonical" href="{{ . }}" />
```

Comment:
`<link rel="canonical" ...>` refers to the page it's on, if I understand correctly. So you're saying that the canonical link for the *landing page* is the PDF, no?

Reply:
Hmm, right, I misunderstood this. For PDFs, we'd have to use the sitemap or HTTP headers.